cmd/spiderpool-init/cmd/client.go

```diff
@@ -60,14 +60,9 @@ func (c *CoreClient) WaitForCoordinatorCreated(ctx context.Context, coord *spide
 			return nil
 		}
 
-		select {
-		case <-ctx.Done():
-			return ctx.Err()
-		default:
-			interval := retryIntervalSec * time.Second
-			logger.Sugar().Infof("Failed to create default Coordinator %s, recreate in %s: %v", coord.Name, interval, err)
-			time.Sleep(interval)
-		}
+		interval := retryIntervalSec * time.Second
```

Comment:
question:
1 what if the user does not use --wait option for helm, is there any timeout for the init pod ? 
2 the init pod should tend to finish its job as much as possible even if the image fails to pull

Reply:
1. No, init-pod will never stop, but the helm installation would fail even with the no --wait option.